<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_product_window</class>
 <widget class="QDialog" name="add_product_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>460</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>506</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog #add_product_window {
	background-image : url(:/Images/addProductWindowBackground.png);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">
font-family: 'Fjalla One', sans-serif;
font-size: 18px;
letter-spacing: 0px;
word-spacing: 0px;
color: #2A2F4F;
font-weight: bold;
text-decoration: none;
font-style: normal;
font-variant: normal;
text-transform: none;
padding:10px 20px;</string>
       </property>
       <property name="text">
        <string>Add Product</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_16">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="styleSheet">
                <string notr="true">font-family: Calibri;
font-size: 12px;
color: #205c6f;
font-weight: bold;
text-transform: none;</string>
               </property>
               <property name="text">
                <string>Name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="LE_name">
               <property name="styleSheet">
                <string notr="true">QLineEdit {
	background-color: #F3E8FF;
	border-radius:8px;
	border:1px solid #ced1e7;
	display:inline-block;
	cursor:text;
	color : #2c2c3b;
	font-family:Impactl;
	font-size:10px;
	font-weight:bold;
	padding:10px 10px;
}
</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_9">
               <property name="styleSheet">
                <string notr="true">font-family: Calibri;
font-size: 12px;
color: #205c6f;
font-weight: bold;
text-transform: none;</string>
               </property>
               <property name="text">
                <string>Price</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="LE_price">
               <property name="styleSheet">
                <string notr="true">QLineEdit {
	background-color: #F3E8FF;
	border-radius:8px;
	border:1px solid #ced1e7;
	display:inline-block;
	cursor:text;
	color : #2c2c3b;
	font-family:Impactl;
	font-size:10px;
	font-weight:bold;
	padding:10px 10px;
}
</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="styleSheet">
                <string notr="true">font-family: Calibri;
font-size: 12px;
color: #205c6f;
font-weight: bold;
text-transform: none;</string>
               </property>
               <property name="text">
                <string>SKU</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="LE_SKU">
               <property name="styleSheet">
                <string notr="true">QLineEdit {
	background-color: #F3E8FF;
	border-radius:8px;
	border:1px solid #ced1e7;
	display:inline-block;
	cursor:text;
	color : #2c2c3b;
	font-family:Impactl;
	font-size:10px;
	font-weight:bold;
	padding:10px 10px;
}
</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="LB_nameError">
             <property name="styleSheet">
              <string notr="true">font-family: Calibri;
font-size: 10px;
color: #820000;
font-weight: bold;
text-transform: none;</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="LB_priceError">
             <property name="styleSheet">
              <string notr="true">font-family: Calibri;
font-size: 10px;
color: #820000;
font-weight: bold;
text-transform: none;</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="LB_skuError">
             <property name="styleSheet">
              <string notr="true">font-family: Calibri;
font-size: 10px;
color: #820000;
font-weight: bold;
text-transform: none;</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="styleSheet">
              <string notr="true">font-family: Calibri;
font-size: 12px;
color: #205c6f;
font-weight: bold;
text-transform: none;</string>
             </property>
             <property name="text">
              <string>Brand</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="LE_brand">
             <property name="styleSheet">
              <string notr="true">QLineEdit {
	background-color: #F3E8FF;
	border-radius:8px;
	border:1px solid #ced1e7;
	display:inline-block;
	cursor:text;
	color : #2c2c3b;
	font-family:Impactl;
	font-size:10px;
	font-weight:bold;
	padding:10px 10px;
}
</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="LB_brandError">
             <property name="styleSheet">
              <string notr="true">font-family: Calibri;
font-size: 10px;
color: #820000;
font-weight: bold;
text-transform: none;</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="styleSheet">
              <string notr="true">font-family: Calibri;
font-size: 12px;
color: #205c6f;
font-weight: bold;
text-transform: none;</string>
             </property>
             <property name="text">
              <string>Category</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="LE_category">
             <property name="styleSheet">
              <string notr="true">QLineEdit {
	background-color: #F3E8FF;
	border-radius:8px;
	border:1px solid #ced1e7;
	display:inline-block;
	cursor:text;
	color : #2c2c3b;
	font-family:Impactl;
	font-size:10px;
	font-weight:bold;
	padding:10px 10px;
}
</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="LB_categoryError">
             <property name="styleSheet">
              <string notr="true">font-family: Calibri;
font-size: 10px;
color: #820000;
font-weight: bold;
text-transform: none;</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_16">
             <property name="styleSheet">
              <string notr="true">font-family: Calibri;
font-size: 12px;
color: #205c6f;
font-weight: bold;
text-transform: none;</string>
             </property>
             <property name="text">
              <string>In Stock</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="LE_stock">
             <property name="styleSheet">
              <string notr="true">QLineEdit {
	background-color: #F3E8FF;
	border-radius:8px;
	border:1px solid #ced1e7;
	display:inline-block;
	cursor:text;
	color : #2c2c3b;
	font-family:Impactl;
	font-size:10px;
	font-weight:bold;
	padding:10px 10px;
}
</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="LB_stockError">
             <property name="styleSheet">
              <string notr="true">font-family: Calibri;
font-size: 10px;
color: #820000;
font-weight: bold;
text-transform: none;</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_15">
             <property name="styleSheet">
              <string notr="true">font-family: Calibri;
font-size: 12px;
color: #205c6f;
font-weight: bold;
text-transform: none;</string>
             </property>
             <property name="text">
              <string>Available</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="LE_available">
             <property name="styleSheet">
              <string notr="true">QLineEdit {
	background-color: #F3E8FF;
	border-radius:8px;
	border:1px solid #ced1e7;
	display:inline-block;
	cursor:text;
	color : #2c2c3b;
	font-family:Impactl;
	font-size:10px;
	font-weight:bold;
	padding:10px 10px;
}
</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="LB_availableError">
             <property name="styleSheet">
              <string notr="true">font-family: Calibri;
font-size: 10px;
color: #820000;
font-weight: bold;
text-transform: none;</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_17">
             <property name="styleSheet">
              <string notr="true">font-family: Calibri;
font-size: 12px;
color: #205c6f;
font-weight: bold;
text-transform: none;</string>
             </property>
             <property name="text">
              <string>Unit</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="LE_unit">
             <property name="styleSheet">
              <string notr="true">QLineEdit {
	background-color: #F3E8FF;
	border-radius:8px;
	border:1px solid #ced1e7;
	display:inline-block;
	cursor:text;
	color : #2c2c3b;
	font-family:Impactl;
	font-size:10px;
	font-weight:bold;
	padding:10px 10px;
}
</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="LB_unitError">
             <property name="styleSheet">
              <string notr="true">font-family: Calibri;
font-size: 10px;
color: #820000;
font-weight: bold;
text-transform: none;</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>4</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font-family: Calibri;
font-size: 12px;
color: #205c6f;
font-weight: bold;
text-transform: none;</string>
             </property>
             <property name="text">
              <string>Added Date</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="DE_addedDate">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_15">
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Maximum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>15</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QCheckBox" name="CB_expCheckBox">
               <property name="styleSheet">
                <string notr="true">font-family: Calibri;
font-size: 12px;
color: #205c6f;
font-weight: bold;
text-transform: none;</string>
               </property>
               <property name="text">
                <string>Add expration date</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="VL_expDate">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="LB_expDate">
               <property name="styleSheet">
                <string notr="true">font-family: Calibri;
font-size: 12px;
color: #205c6f;
font-weight: bold;
text-transform: none;</string>
               </property>
               <property name="text">
                <string>Expration Date</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="DE_exDate">
               <property name="minimumSize">
                <size>
                 <width>250</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>250</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="styleSheet">
              <string notr="true">font-family: Calibri;
font-size: 12px;
color: #205c6f;
font-weight: bold;
text-transform: none;</string>
             </property>
             <property name="text">
              <string>Description</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="TE_description">
             <property name="styleSheet">
              <string notr="true">QTextEdit {
	background-color: #F3E8FF;
	border-radius:8px;
	border:1px solid #ced1e7;
	display:inline-block;
	cursor:text;
	color : #2c2c3b;
	font-family:Impactl;
	font-size:10px;
	font-weight:bold;
	padding:10px 10px;
}
</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_14">
           <property name="spacing">
            <number>4</number>
           </property>
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Minimum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>15</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="PB_addProduct">
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
    background-color: #C7E9B0;
    border: 1px solid #CCCCCC;
    border-radius: 8px;
    padding: 20px 20px;
	color: #1c1d06;
	font-family : Cabin;
	font-weight : bold;
	font-size : 12px;

	
}

QPushButton::hover {
	background-color : #eff4fc;

}</string>
             </property>
             <property name="text">
              <string>Add Product</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Minimum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>10</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="PB_cancel">
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
    background-color: #f4c3d6;
    border: 1px solid #CCCCCC;
    border-radius: 8px;
    padding: 20px 20px;
	color: #1c1d06;
	font-family : Cabin;
	font-weight : bold;
	font-size : 12px;

	
}

QPushButton::hover {
	background-color : #eff4fc;

}</string>
             </property>
             <property name="text">
              <string>Cancel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
